docs/code_contribution_guidelines.md

```diff
@@ -429,17 +483,27 @@ master.  In certain cases the code reviewer(s) or interested committers may help
 you rework the code, but generally you will simply be given feedback for you to
 make the necessary changes.
 
+During the process of responding to review comments, we prefer that changes be
+made with [fixup commits](https://robots.thoughtbot.com/autosquashing-git-commits). 
+The reason for this is two fold: it makes it easier for the reviewer to see
+what changes have been made between versions (since Github doesn't easily show
+prior versions like Critique) and it makes it easier on the PR author as they
+can set it to auto squash the fix up commits on rebase.
```

Comment:
During autosquash, conflicts can still arise. Would it be better to already order the fixup commits, but not squash yet, so any conflicts are already resolved?

Reply:
Ah, you mean order them immediately following the commit they fix? 

I have never tried that, but that sounds like a pretty good idea. Might even make them easier to review?